docs/hassbian_config.md

```diff
@@ -0,0 +1,31 @@
+## Description
```

Comment:
This should be extended a bit.
Documentation about the flags like `-y` & `-t` are missing.

Reply:
When this PR was created, those flags did not exist :)
They are added now :)